### PR TITLE
Add config to mount a scratch directory

### PR DESCRIPTION
**Pull request by rendhalver**
This adds an optional config to mount a scratch directory into your ocm container.

### CODE REVIEW

File: ocm-container.sh [hunk 1]
```diff
@@ -120,6 +120,12 @@ if [ -d "${OPS_UTILS_DIR}" ]; then
 "
 fi
 
+### mount a scratch dir
+if [ -n "$SCRATCH_DIR" ]
+then
+  SCRATCH_DIR_MOUNT="-v ${SCRATCH_DIR}:/home/scratch/"
```
Comment (clcollins) [inline ocm-container.sh:126]: Is this intended to be `/home/scratch` or `~/scratch` (/home/$USER/scratch)?

**Review by clcollins — COMMENTED** [review]
One question about the mount point inside the container, but otherwise, LGTM.

Can you confirm where the scratch dir should be mounted?

File: ocm-container.sh (hunk 1 above)
Comment (rendhalver) [reply ocm-container.sh:126]: I mounted it there locally for a reason I can't remember.
I guess `~/scratch` makes more sense. We are mounting the utils dir under `~/` so let's move it there.
I will update the branch.

**Pull request merged**